\begin{center}
    \large\textsc{Abstract}
\end{center}
\markboth{Abstract}{}
\addcontentsline{toc}{section}{Abstract}

During human brain development, cells undergo remarkable fate transitions to give rise to a vast diversity of cell types. This process is tightly coordinated by a complex regulatory landscape that guides cells towards their terminal fate. Exploring the molecular mechanisms that shape this landscape in humans has long remained challenging due to both ethical and technical reasons. Two converging technologies are now transforming the field and offer exciting new possibilities to study human brain development: \textit{In vitro} brain models grown from human stem cells and single-cell genomics. 

In this thesis, we want to approach the study of human brain developement with fresh eyes, by harnessing and connecting this new set of technologies. Towards this goal, we use brain organoids to model regional diversification during early human brain development and apply modern single-cell genomics methods and perturbation experiments to probe them at the level of individual cells. In the first part, we address the challenge of systematically and reliably characterizing organoid phenotypes accross protocols and experients. For this, we develop a new computational tool, called VoxHunt, to assess the regional composition of brain organoids by mapping them to large reference atlases. In the second part, we sought to illuminate the gene regulatory events shaping brain region diversification. We acquire a single-cell transcriptome and chromatin accessibility data over a dense time course, covering organoid development from pluripotency up until neurogenesis. We develop Pando, a flexible framework for inferring gene regulatory networks (GRNs) from multi-modal datasets. Using Pando, we infer a global GRN underlying human brain organoid development. We use pooled genetic perturbations to assess transcription factor requirement for brain region diversification and show that the transcription factor GLI3 is required for cortical fate establishment in humans. Using the inferred GRN in conjuntion with transcriptome and chromatin accessibility profiles of GLI3-perturbed cells, we identify two distinct genetic networks central to telencephalic phate decisions. After having explored gene regulatory underpinnings of brain development, in the third part we were curious how they are altered in a disease context. Here, we perform a pooled genetic screen of high-risk autism spectrum disorder (ASD) genes to investigate the regulatory roots of Autism. We show that many genes play a role in region establishment and cell fate determination and discover developmental stages suscepible to ASD gene perturbations. With the help of a GRN inferred by Pando, we further identify ASD-specific and perturbation-enriched regulatory modules. We find that perturbation of the BAF subunit ARID1B causes dysregulation of OLIG1 and leads to increased fate transitions towards oligodendrocyte precursor cells. In the last part, we shed further light on the epigenetic mechanisms coordinating brain development. We profile three histone modifications and mRNA expression in single cells over a time course of brain and retina organoid developement from pluripotency. Using this data, we reconstruct the epigenomic trajectories governing cell fate acquisition. We find that epigenetic switches control the expression of major cell fate regulators and that epigenetic activation precedes gene expression during neurogenesis. Finally, we chemically perturb the H3K27me3 writer complex and show that H3K27me3-mediated repression is required to stabilize fate commitment during early organoid development. 

In summary, we here present new methods and insights to help illuminate the regulatory landscape underlying human brain development. We also provide valuable multi-omic reference atlases of brain organoid development and pioneer the application of perturbation screens in organoids to systematically interrogate gene regulation. Altogether, we hope that the contributions of this thesis provide a framework for how human organoid systems and single-cell technologies can be leveraged to reconstruct human developmental biology.


