\begin{center}
    \large\textsc{Abstract}
\end{center}
\markboth{Abstract}{}
\addcontentsline{toc}{section}{Abstract}

During human brain development, cells undergo remarkable fate transitions to give rise to a vast diversity of cell types. This process is tightly coordinated by a complex regulatory landscape that guides cells towards their terminal fate. It has long remained challenging to explore the molecular mechanisms that shape this landscape in humans. The convergence of two new technologies now offers fresh possibilities to study human brain development: \textit{In vitro} brain models grown from human stem cells and single-cell genomics. 

In this thesis, we use brain organoids to model cell fate acquisition during early human brain development and probe them using modern single-cell technologies. In the first part, we address the challenge of systematically characterizing organoid phenotypes. For this, we develop VoxHunt, a novel computational tool to assess the regional composition of brain organoids by mapping them to large reference atlases. In the second part, we illuminate the gene regulatory events shaping brain region diversification. We generate a single-cell transcriptome and chromatin accessibility atlas over a dense time course of early organoid development. We develop Pando, a flexible framework for inferring gene regulatory networks (GRNs) from multi-modal datasets and use it to infer a global GRN underlying human brain organoid development. We apply pooled genetic perturbations to assess transcription factor requirement for brain region diversification and show that the GLI3 is required for cortical fate establishment in humans. In the third part we were curious how brain development is altered in autism spectrum disorder (ASD) and performed a pooled genetic screen of ASD risk genes. We show that perturbations in many ASD genes lead to alterations of cell fate establishment and identify an ASD-specific sub-GRN. Specifically, we find that perturbation of ARID1B leads to aberrant fate transitions towards oligodendrocyte precursor cells. In the last part, we extend our previous multiomic atlas of early human brain development to shed light on epigenetic mechanisms. We profile three histone modifications and mRNA expression in single cells over an organoid developmental time course and identify epigenetic switches controlling cell fate bifurcations. Through perturbation of epigenetic modulators, we further show that epigenetic regulation is required to stabilize fate commitment during early organoid development. 

In summary, we here present new methods and insights to help illuminate the regulatory landscape underlying human brain development. We also provide valuable multi-omic reference atlases of brain organoid development and pioneer the application of perturbation screens in organoids to systematically interrogate gene regulation. Altogether, this thesis provides a framework for how human organoid systems and single-cell technologies can be leveraged to reconstruct human developmental biology.


